internal/node/config/config.go

```diff
@@ -290,6 +292,34 @@ func (n *NodeConfig) FrontendDev() bool {
 	return viper.GetBool("frontend_dev")
 }
 
+func (n *NodeConfig) DefaultApps() []*types.PostAppRequest {
+	defaultApps := viper.GetStringMap("default_apps")
+	var appRequests []*types.PostAppRequest
+
+	for _, appConfig := range defaultApps {
+		// Marshal the app config into bytes, and then unmarshal into a PostAppRequest
+		appConfigBytes, err := yaml.Marshal(appConfig)
+		if err != nil {
+			log.Error().Err(err).Interface("app", appConfig).Msg("Failed to marshal default app config")
+			continue
+		}
+		log.Info().Str("app", string(appConfigBytes)).Msg("Default app config")
+
+		var request types.PostAppRequest
```

Comment:
can we go directly from string -> types.PostAppRequest?

the `Marshal` and `Unmarshal` seems unnecessary and less efficient

Reply:
I found there's a way to user Viper to unmarshal directly